### ISSUE TITLE
Compatibility problems with other gems for jekyll-auth plugin

### ISSUE THREAD

**Issue by JohnRPB** [benbalter/jekyll-auth#95]
Hi there,

I'm new to Jekyll and in the `bundle install` portion of the setup I get errors saying that the bundler cannot find any compatible versions of several gems. To my untrained eyes, it looks like I may have to force the installation of a previous version of jekyll (although there seem to be conflicts with multiple versions?).

Anyway, here's the error message from the terminal after running `bundle update`, producing basically the same result as `install`:

```{bash}
Bundler could not find compatible versions for gem "colorator":
  In Gemfile:
    jekyll (= 3.3.0) was resolved to 3.3.0, which depends on
      colorator (~> 1.0)

    jekyll-auth was resolved to 2.0.0, which depends on
      colorator (~> 0.1)
Bundler could not find compatible versions for gem "jekyll":
  In Gemfile:
    jekyll (= 3.3.0)

    jekyll-auth was resolved to 0.1.0, which depends on
      github-pages was resolved to 1, which depends on
        jekyll (= 1.1.2)

    jekyll-feed (~> 0.6) was resolved to 0.8.0, which depends on
      jekyll (~> 3.3)
Bundler could not find compatible versions for gem "safe_yaml":
  In Gemfile:
    jekyll (= 3.3.0) was resolved to 3.3.0, which depends on
      safe_yaml (~> 1.0)

    jekyll-auth was resolved to 0.2.0, which depends on
      safe_yaml (~> 0.9.0)

```

Thanks for taking a look at this,

 - Johann

**Comment by asanger** [comment]
hi @JohnRPB - this has been fixed with #96, so the next release of jekyll-auth will correct your issue.

In the mean time, you've got a couple options:
1. Downgrade jekyll to v3.0.0.
2. Specify the repository in your gemfile like so:
```ruby
gem "jekyll-auth", git: "git@github.com:benbalter/jekyll-auth.git"
```

**Comment by benbalter** [comment]
https://github.com/benbalter/jekyll-auth/releases/tag/v2.1.0
